login-utils/login.c

```diff
@@ -1260,6 +1259,7 @@ int main(int argc, char **argv)
 	char *buff;
 	int childArgc = 0;
 	int retcode;
+	unsigned int timeout = LOGIN_TIMEOUT;
```

Comment:
nitpicking ... maybe the initialization is unnecessary as we call getlogindefs_num("LOGIN_TIMEOUT" ...)

Reply:
Removed in 0b4d75fae55b4a5ff8f65df8551c56cf1eeb9b08.